firefox_ui_harness/options/functional.py

```diff
@@ -0,0 +1,18 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import firefox_puppeteer
+import firefox_ui_tests
+
+from . import FirefoxUIOptions
+
+
+class FunctionalOptions(FirefoxUIOptions):
+
+    def parse_args(self, *args, **kwargs):
+        options, tests = FirefoxUIOptions.parse_args(self, *args, **kwargs)
+
+        tests = [firefox_puppeteer.manifest, firefox_ui_tests.manifest_functional]
```

Comment:
We already have the ability to specify tests from the command line. These entry points don't provide anything further. This is a configuration change, it doesn't need to be a code change.